### PR TITLE
Completed schema.sql file to included hard-coded reviews for product 1, refactored seeder to not generate random reviews for product id: 1

### PR DESCRIPTION
**Pull request by ikheraj17**

### CODE REVIEW

**Comment by Joseph765** [comment]
LGTM! I can't see any issues your code seems really clean.

File: server/schema.sql [hunk 1]
```diff
@@ -6,11 +6,31 @@ USE reviewList;
...
+
+INSERT INTO reviews (id, username, rating, reviewDate, review, productID, shopID) VALUES (7, 'Ash Ketchum', 2, "2020-04-20", 'I bought this so I could keep cathching pokemon during this crisis. Unfortunately, the durablity of this mask is not great... \nA pidgey pecked right through this in a matter of seconds. ', 1, 1);
+
+INSERT INTO reviews (id, username, rating, reviewDate, review, productID, shopID) VALUES (8, 'Ronnie Coleman', 5, "2020-04-30", 'This mask is straight up LIGHT WEIGHT. I wear it when I workout to look super cool. \n I would recommend this product.', 1, 1);
+
+INSERT INTO reviews (id, username, rating, reviewDate, review, productID, shopID) VALUES (9, 'Seth Rogen', 4, "2020-04-27", 'Insert signature Seth laugh here. This mask is awesome! \nMy one gripe is that I wish there were alternate sizes for my large head.', 1, 1);
+
+INSERT INTO reviews (id, username, rating, reviewDate, review, productID, shopID) VALUES (10, 'Batman', 3, "2020-04-22", 'This mask helps me fight crime in Gotham while maintaining a certain standard of hygeine. It flies off sometimes when I dive off skyscrapers, but still is good for the price.', 1, 1);
```
Comment (mta630) [inline server/schema.sql:36]: I love the reviews you wrote for product 1

File: server/seeder.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  connection.query(queryStr, params, (err) => {
+    if (err) {
+      throw (err);
+    }
+  });
+  // eslint-disable-next-line no-console
+}
+
+connection.end();
```
Comment (mta630) [inline server/seeder.js:43]: 10k should do it

**Review by mta630 — APPROVED** [review]
LGTM

File: server/schema.sql [hunk 3]
```diff
@@ -6,11 +6,31 @@ USE reviewList;
...
+  reviewDate DATE NOT NULL,
+  review varchar(500) default '' NOT NULL,
+  productID integer default 0 NOT NULL,
+  shopID integer default 1 NOT NULL,
   PRIMARY KEY(id)
-);
\ No newline at end of file
+);
+
+INSERT INTO reviews (id, username, rating, reviewDate, review, productID, shopID) VALUES (1, 'John Jacob', 5, "2020-04-21", 'This mask is literally the best thing I have purchased in my entire 40 years of life.\nJust buy it...NOW', 1, 1);
```
Comment (Joseph765) [inline server/schema.sql:18]: Good job!

**Review by Joseph765 — COMMENTED** [review]
Great!

**Pull request merged**